collections/SubredditLists.js

```diff
@@ -1,7 +1,20 @@
-import { fetchAll } from '../apiBase/APIResponsePaging';
+import { fetchAll, afterResponse, beforeResponse } from '../apiBase/APIResponsePaging';
 import Listing from './Listing';
 import SubredditEndpoint from '../apis/SubredditEndpoint';
 
+// subreddit's uuid is the clean name but for paging we need the thing id
+export const subredditAfter = (apiResponse) => {
+  const after = afterResponse(apiResponse);
```

Comment:
🔕 here and everywhere else, it'd be nice if this was `subredditAfterId`, `afterResponseId`, etc - so you know it's for getting an id. it's a little ambiguous otherwise. Not a change for this PR, though.